networkit/centrality.pyx

```diff
@@ -2928,3 +2932,120 @@ class SciPyPageRank(SpectralCentrality):
 
 		self.eigenvector = spectrum[1][0]
 		self.eigenvalue = spectrum[0][0]
+
+##### from branch ComplexPaths (K. Ahrens)
+
+cdef extern from "<networkit/centrality/ComplexPaths.hpp>" namespace "NetworKit::ComplexPathAlgorithm":
+
+	cpdef enum _Mode  "NetworKit::ComplexPathAlgorithm::Mode":
+		singleNode,
+		allNodes
+
+cdef class ComplexPathMode(object):
+	singleNode = _Mode.singleNode
+	allNodes   = _Mode.allNodes
```

Comment:
Enums in Python are written in capital letters with underscore (SREAMING_SNAKE_CASE): `SINGLE_NODE` and `ALL_NODES`